modin/experimental/engines/pyarrow_on_ray/frame/axis_partition.py

```diff
@@ -19,23 +21,45 @@
 
 
 class PyarrowOnRayFrameAxisPartition(BaseFrameAxisPartition):
+    """
+    Class defines axis partition interface with PyArrow backend and Ray engine.
+
+    Inherits functionality from ``BaseFrameAxisPartition`` class.
+
+    Parameters
+    ----------
+    list_of_blocks : list
+        List with partition objects to create common axis partition for.
+    """
+
     def __init__(self, list_of_blocks):
         # Unwrap from BaseFramePartition object for ease of use
         self.list_of_blocks = [obj.oid for obj in list_of_blocks]
 
     def apply(self, func, num_splits=None, other_axis_partition=None, **kwargs):
-        """Applies func to the object in the plasma store.
-
+        """
+        Apply func to the object in the Plasma store.
+
+        Parameters
+        ----------
+        func : callable or ray.ObjectRef
+            The function to apply.
+        num_splits : int, optional
+            The number of times to split the resulting object.
+        other_axis_partition : PyarrowOnRayFrameAxisPartition, optional
+            Another ``PyarrowOnRayFrameAxisPartition`` object to apply to
+            `func` with this one.
+        **kwargs : dict
+            Additional keyward arguments to pass with `func`.
+
+        Returns
+        -------
+        list
+            List with ``RayRemotePartition`` objects.
```

Comment:
@amyskov , we do not have "RayRemotePartition" objects. I guess you meant "PyarrowOnRayFramePartition". I see you were using "RayRemotePartition" in multiple places that needs to be corrected. Can you fix this as part of another issue?

Reply:
> Can you fix this as part of another issue?

Clarification - "another issue" means "separate issue specifically created to rectify this", not "some other random issue" 😉

Reply:
Ok, issue was created https://github.com/modin-project/modin/issues/3148.